src/ape_plugins/_cli.py

```diff
@@ -42,17 +47,19 @@ def plugins_argument():
     """
 
     def callback(ctx, param, value: Tuple[str]):
+
         if not value:
             ctx.obj.abort("You must give at least one requirement to install.")
 
         elif len(value) == 1 and Path(value[0]).resolve().exists():
             # User passed in a path to a config file.
             config_path = Path(value[0]).expanduser().resolve()
+
             if config_path.name != CONFIG_FILE_NAME:
                 config_path = config_path / CONFIG_FILE_NAME
 
             config = load_config(config_path)
-            plugins = config.get("plugins") or []
+            plugins = config.get("plugins", [])
```

Comment:
Hey. I would advise not making this change.

There is actually a difference here between what it was and what you are changing it too.

The default argument in `get()` only gets returns *if they key is not present in the dictionary*.

In the case where they key _is_ present in the dictionary, it always uses that, even if that key is `None`.

The way it was before, all falsey values get coerced to the empty list. So even if they key exists but its value is `None`, it would get turned into an empty list.

the change you are making breaks that safety!

Reply:
Thanks for breaking it down for me, that makes sense!